data/libs/NameGen.lua

```diff
@@ -195,11 +224,20 @@ NameGen.orbitalStarportFormats = {
 	"{name} Dock",
 	"{name} Depot",
 	"{name} Anchorage",
+	"{name} Junction",
+	"{name} Connection",
+	"{name} Academy",
+	"{name} University",
+	"{name} Waystation",
+	"{name} Laboratory",
+	"{name} Station{number}",
+	"{name} Citadel",
 }
 
 NameGen.surfaceStarportFormats = {
-	"{name}",
-	"{name}",
+	"{name}{number}",
+	"{name}{number}",
+	"{name}{number}",
```

Comment:
Note that this produces an exceptionally high number of single-word starport names.

![image](https://github.com/user-attachments/assets/e76f32ab-a59e-4404-927c-d985890021fa)
(Taiton is not the direct result of random generation, all other names are as-generated.)

This isn't a problem with long multi-syllable names, but a station named *only* "Lloyd" (or even "Rothmann") is quite immersion-breaking. I will most likely address this in a follow-on PR - I'm happy with the additional diversity of suffixes this PR adds and it's been sitting open long enough already.

Reply:
Hm, **Taiton**? I don't see the same list of stations for NN 3737.

> a station named _only_ "Lloyd" (or even "Rothmann") is quite immersion-breaking.

I think rather the opposite. Too many station names with a name and a random 'Spaceport' stuck onto it is more immersion breaking for me. Details like that isn't of great importance to me though. Thanks for the merge!

> I will most likely address this in a follow-on PR

Please tag me... :)

Reply:
> Hm, **Taiton**? I don't see the same list of stations for NN 3737.

Yeah, I clicked randomize a bunch in the system editor to get an idea of what the distribution of names was.

> I think rather the opposite. Too many station names with a name and a random 'Spaceport' stuck onto it is more immersion breaking for me. Details like that isn't of great importance to me though. Thanks for the merge!
> 
> Please tag me... :)

The thing here that's immersion breaking (at least in English speaking / British/American style countries) is that towns/cities can have a single "person name" as their name, but transit infrastructure (airports, metro stations, etc) are almost always named e.g. `<Area name> Airport` or `<Area name> Station`. For example, it's "John F. Kennedy International Airport" or "Dallas-Fort Worth Intl. Airport", not "John F. Kennedy" or "Dallas-Fort Worth".

Given that these names are assigned to the spaceport specifically (an expectation set by the icon) there's an "uncanny valley" feeling when looking at those names, especially next to a name that does follow convention.

Perhaps it might help your sense of immersion if we were to generate some `<First Name> <Last Name> Spaceport` style name to reduce the deluge of `<Name> Spaceport`s?